content/3_Silver/Binary_Search.mdx

```diff
@@ -540,34 +531,36 @@ public class BinarySearch{
 <PySection>
 
 ```python
-# checks whether the number of given operations is sufficient
-# to raise the median of the array to x
-def f(x):
-	operationsNeeded = 0
-	for i in range((n-1)//2,n):
-		operationsNeeded += max(0, x-v[i])
-	if operationsNeeded <= k:
-		return True
-	else:
-		return False
+from typing import Callable
+
+
```

Comment:
```suggestion
```

Reply:
ryan did you not see my comments on the previous pr